crates/binstalk-downloader/Cargo.toml

```diff
@@ -37,7 +37,7 @@ tokio-tar = "0.3.0"
 tokio-util = { version = "0.7.8", features = ["io"] }
 tracing = "0.1.37"
 # trust-dns-resolver must be kept in sync with the version reqwest uses
-trust-dns-resolver = { version = "0.23.0", optional = true, default-features = false, features = ["dnssec-ring"] }
+trust-dns-resolver = { version = "0.22.0", optional = true, features = ["dnssec-ring"] }
```

Comment:
In case you're wondering why I did this: without this change, the `tokio-runtime` feature is disabled which means we don't have access to `TokioAsyncResolver`.